### DISCUSSION TITLE
Keycloak Operator - roadmap?

### DISCUSSION THREAD

**Discussion by sventorben** [keycloak/keycloak#27144] [Q&A] [answered]
Hello! I basically have a similiar question than this one: https://github.com/keycloak/keycloak/discussions/26448

What is the plan to support not only creation of realms but also management of other objects such as clients, scopes, roles, users, policies etc.?

The current operator seems useless in our scenarios where we would like to maintain lifecycle of object with an GitOps approach. Maybe we are missing something. Are there any more features available than what is described in the quick start guides?

**Comment by vmuzikar** [comment, marked as answer]
@sventorben Thank you for your interest in the Keycloak Operator. We do see the importance of a GitOps-like approach as it's native to K8s environment. Adding more CRs is still [planned](https://github.com/keycloak/keycloak/issues/13566) (probably just Clients for now as they make most sense) but I can't share any ETA at the moment as we're also struggling with other priorities. We hope it'll happen this year, though.
